### PR TITLE
finished react1 week1 homework

### PR DESCRIPTION
**Pull request by chunmeishui**

### CODE REVIEW

File: react/react1/week1/todo-list/src/App.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import './App.css';
+import { MainToDo } from './MainToDo';
+import { TodoData } from './TodoData';
+function App() {
+  return (
```
Comment (senner007) [inline react/react1/week1/todo-list/src/App.js:5]: Install the prettier vscode extension and format your code

File: react/react1/week1/todo-list/src/TodoItems.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react"
+
+export function TodoItems(props){
```
Comment (senner007) [inline react/react1/week1/todo-list/src/TodoItems.js]: TodoItem singular and not TodoItem**s**, right?

File: react/react1/week1/todo-list/src/TodoList.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import {TodoItems} from "./TodoItems";
+import React from "react";
+export function TodoList(props) {
+    return (
+        <ul>
```
Comment (senner007) [inline react/react1/week1/todo-list/src/TodoList.js]: Again, I prefer not to inline the map operations in the JSX. Assign it to a const before the return and give it a meaningful name

**Review by senner007 — APPROVED** [review]
Great! 👍

File: react/react1/week1/todo-list/src/App.js (hunk 1 above)
Comment (chunmeishui) [reply react/react1/week1/todo-list/src/App.js:5]: okay,Thanks :)

File: react/react1/week1/todo-list/src/TodoItems.js (hunk 2 above)
Comment (chunmeishui) [reply react/react1/week1/todo-list/src/TodoItems.js]: yes, right.I thought it will dispaly lots of items.so I named it as todoItems.;)

**Pull request merged**